frontend/tests/jest/utils.tsx

```diff
@@ -157,3 +163,29 @@ export const getProductWithWarranty = (
 
    return product;
 };
+
+export const getReviewsResponse = (
+   type: 'filled' | 'empty' = 'filled'
+): ProductReviewData => {
+   let reviews: ProductReview[] = mockedReviews;
+
+   if (type === 'empty') {
+      reviews = [];
+   }
+
+   return {
+      reviews: reviews,
+      count: reviews.length,
+      average:
+         // @ts-ignore
+         reviews.reduce((total, next) => total + next.rating, 0) /
+         (reviews.length == 0 ? 1 : reviews.length),
```

Comment:
This is a minor point and I don't think you need to change it just for this, but I think when handling the empty case to avoid dividing by 0, it makes more sense to put the whole reduce in the non-zero case rather than just conditionalizing the divisor when the dividend is also going to be meaningless.
```suggestion
         reviews.length == 0 ? 0 : reviews.reduce((total, next) => total + next.rating, 0) / reviews.length,
```

Reply:
Thanks @sterlinghirsh for great the suggestion! Updated in https://github.com/iFixit/react-commerce/pull/1173/commits/2c391f6f02c82b8e20474529de48350a679c8cdc.